internal/prompt/prompt.go

```diff
@@ -129,9 +129,7 @@ func (p *Prompt) InputAndValidate(title, message string, defaultResponse *string
 		if nonInteractiveResponse == nil {
 			return "", ErrNoForceOption
 		}
-	}
-
-	if !p.isInteractive {
+	} else if !p.isInteractive {
```

Comment:
The bug here is that we were checking for force and setting `nonInteractiveResponse` but if `!p.isInteractive` is also true (it's usually not), then `nonInteractiveResponse` would be overwritten with a different value.